core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -395,12 +394,18 @@ public synchronized void flush() throws IOException, JournalClosedException {
 
   @Override
   public synchronized void close() throws IOException {
-    Closer closer = Closer.create();
+    if (mClosed) {
+      return;
+    }
+    mGarbageCollector.close();
```

Comment:
Is it possible for an exception to be thrown here?

Reply:
Highly unlikely. It's completely owned by us with no 2rd party code and I left it out of try in order to catch potential bugs in the future.